src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/ISymbolNode.cs

```diff
@@ -12,6 +12,7 @@ public interface ISymbolNode
     {
         void AppendMangledName(NameMangler nameMangler, Utf8StringBuilder sb);
         int Offset { get; }
+        bool RepresentsIndirectionCell { get; }
```

Comment:
I think the name is fine. A comment explaining to future consumers of the interface what implications this has on the code-generator consuming the symbol would be nice.